config/zones/US-HI-OA.yaml

```diff
@@ -4,14 +4,14 @@ bounding_box:
   - - -157.14716739499988
     - 22.209850429000085
 capacity:
-  battery storage: 186
+  battery storage: 40.9
```

Comment:
This don't look right, any explanation for the sudden decrease?

(It's possible the source was not added to the sources file.)

Reply:
In #4526 was the Kapolein Energy Storage Facility added. This one is probably missing because it is not yet in operation, according to this source: https://www.spglobal.com/marketintelligence/en/news-insights/latest-news-headlines/us-energy-storage-wave-builds-strength-amid-delays-72885152

This makes sense since the facility isn't yet listed in the EIA data set.

Reply:
I think we could just add 185 MW with the existing EIA value then and hope it updates correctly next year.

Reply:
That's fine with me. The Hawaii data isn't very accurate at the moment anyway. The estimated data is still based on the coal power plant running. However, that's another topic.